synapse/models/inet.py

```diff
@@ -891,6 +893,16 @@ def getBaseModels():
                     ('logout', {'ptype': 'time', 'doc': 'The time the account logged out of the service.'})
                 ]),
 
+                ('inet:web:action', {'ptype': 'inet:web:action'}, [
+                    ('action', {'ptype': 'str:lwr', 'doc': 'The action performed'}),
```

Comment:
@therealsilence - @vEpipiyte proposed that we call the `action` prop `act`. Which do you want to use?

Reply:
Agree with both - "act" shorter than "action", and the in-model docs can explain what the form / props are. And like the idea of "info" as a sort of "catch all" for additional data. 

Semi-related: saw comments in chat about possible "inet:web:actson" xref type, but would rather find a real-world use case before we worry about it.

Reply:
acctinfo / info is a get-out-of-jail json field we will need to have.

Reply:
I strongly suspect we will need to be able to xref other nodes from the inet:web:action form.  Since creating inet:web:actref is easy, i'd say we create it now so we dont have to wait for a release cycle when we first encounter the need?

Reply:
<shrug> I'm neutral...I guess I can see use cases like "this change password action references this "passwd" node" or "this updated profile picture action references this file:bytes"?

Reply:
fixed